scripts/pretraining/pretraining_utils.py

```diff
@@ -56,9 +56,8 @@ def tokenize_lines_to_ids(lines, tokenizer):
     return results
 
 
-def get_all_features(x):
-    """
-    Get the feature data in numpy form.
+def get_all_features(file_list, output_file, tokenizer, max_seq_length, short_seq_prob):
+    """Get the feature data in numpy form.
```

Comment:
This function is used with `pool.imap`, and this modification will raise an error with message `TypeError: get_all_features() missing 4 required positional arguments: 'output_file', 'tokenizer', 'max_seq_length', and 'short_seq_prob'`
 https://github.com/dmlc/gluon-nlp/blob/9e268c0e5e0393e4ce19a4161ef2a2f8eb56797d/scripts/pretraining/data_preprocessing.py#L74

BTW, it is also used in https://github.com/dmlc/gluon-nlp/blob/9e268c0e5e0393e4ce19a4161ef2a2f8eb56797d/scripts/pretraining/pretraining_utils.py#L273-L274

Reply:
Fixed